api-reference/endpoint/get-software.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: "Get Software Info"
+api: "GET https://eol.xeol.com/api/v1/{identifier}"
```

Comment:
a `cpe` or `purl` identifier is not user friendly in this endpoint, since it has has special characters that need to be escaped.

We should allow them to query filter by any of the attributes in entity

```
 "entity": {
    "name": "Red Hat Enterprise Linux",
    "version": "9",
    "released": "2022-05-17",
    "type": "os",
    "vendor": "Red Hat"
  },
```